katas/content/teleportation/Common.qs

```diff
@@ -0,0 +1,7 @@
+namespace Kata.Verification {
+    operation EntangledWrapper_Reference(qs : Qubit[]) : Unit is Adj + Ctl {
```

Comment:
It's convenient to have operation names for wrappers and references to match the name of the original operation with prefixes
```suggestion
    operation EntangleWrapper_Reference(qs : Qubit[]) : Unit is Adj + Ctl {
```